src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipe.cs

```diff
@@ -189,7 +201,7 @@ internal static class EventPipeInternal
         // These PInvokes are used by the configuration APIs to interact with EventPipe.
         //
         [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
-        internal static extern UInt64 Enable(string outputFile, uint circularBufferSizeInMB, long profilerSamplingRateInNanoseconds, EventPipeProviderConfiguration[] providers, int numProviders);
+        internal static extern UInt64 Enable(string outputFile, uint circularBufferSizeInMB, long profilerSamplingRateInNanoseconds, EventPipeProviderConfiguration[] providers, int numProviders, ulong multiFileTraceLengthInSeconds);
```

Comment:
This is OK for now, but as a future edit we really should pass a class/struct containing all this config information (with defaults if you pass null),  That way adding more is easier and the number of args does not get out of control.

Reply:
Agreed.